processor/coralogixprocessor/config.go

```diff
@@ -0,0 +1,36 @@
+package coralogixprocessor // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/coralogixprocessor"
+
+import "fmt"
+
+type CacheConfig struct {
```

Comment:
Does this need to be public?

Reply:
[70e11a5](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33458/commits/70e11a571a3a04acfeb8ee53d885f725c78d8275)